internal/scripts/docker_env/Dockerfile.conda

```diff
@@ -1,9 +1,9 @@
 # Dockerfile to install conda
 
```

Comment:
```suggestion

# Should match the OS used in https://github.com/dtcenter/METbaseimage
```
Might be worth noting the relationship to dtcenter/METbaseimage here. Not sure how specific you want to be though.